umap/static/umap/js/umap.forms.js

```diff
@@ -801,6 +801,23 @@ L.FormBuilder.Range = L.FormBuilder.Input.extend({
   value: function () {
     return L.DomUtil.hasClass(this.wrapper, 'undefined') ? undefined : this.input.value
   },
+
+  buildHelpText: function () {
```

Comment:
I hacked the help text function, I missed some extra helpers to be able to inject some HTML where I want to.